en/orm/entities.rst

```diff
@@ -180,6 +180,57 @@ name will be the lower case and underscored version of the method::
 
 Do bear in mind that virtual fields cannot be used in finds.
 
+Storing explicit NULL values
+----------------------------
+
+By default CakePHP will store empty values such as empty strings as they come in.
```

Comment:
Isn't this behavior only present in string type columns? That might be worth mentioning.